pkg/pillar/hypervisor/nokube.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !kubevirt
+
+package hypervisor
+
+const (
+	KubevirtHypervisorName = "kubevirt"
+)
+
+// newKubevirt in this file is just stub for non-kubevirt hypervisors.
+func newKubevirt() Hypervisor {
```

Comment:
I am curious. With the other hypervisors, we don't have two distinct files with `//go:build` directives. There is no `xen.go` and `noxen.go`, or `kvm.go` and `nokvm.go`. What changes that we need this here?

Reply:
Kubernetes Go packages increase the zedbox (and the eve installer) size by several megabytes.
We want to avoid having it included in kvm and xen EVE variants, where it is not going to be used anyway.

Reply:
Does that hold true for others? Do we want similar disabling?

Reply:
Neither xen nor kvm has any Go dependencies so there would be no value in having them disabled when not needed.